tests/functional/setups/manu_sneddon_2d.py

```diff
@@ -0,0 +1,436 @@
+import numpy as np
+import porepy as pp
+from porepy.viz.data_saving_model_mixin import VerificationDataSaving
+from porepy.applications.convergence_analysis import ConvergenceAnalysis
+from porepy.models.protocol import PorePyModel
+from typing import Literal
+
+
+from dataclasses import dataclass
+
+def compute_eta(pointset_centers: np.ndarray, center: np.ndarray) -> np.ndarray:
+    """
+    Compute the distance fracture points to the fracture centre.
+
+    Parameter::
+    pointset: Array containing coordinates on the fracture
+    center: fracture centre
+
+    Return:
+        Array of distances each point in pointset to the center
+
+    """
+    return pp.geometry.distances.point_pointset(pointset_centers, center)
+
+
+def get_bem_centers(
+    a: float, h: float, n: int, theta: float, center: np.ndarray
+) -> np.ndarray:
+    """
+    Compute coordinates of the centers of the bem segments
+
+    Parameters:
+        a: half fracture length
+        h: bem segment length
+        n: number of bem segments
+        theta: orientation of the fracture
+        center: center of the fracture
+
+    Return:
+        Array of BEM centers
+    """
+
+    # Coordinate system (4.5.1) in page 57 in in book Crouch Starfield 1983 Boundary Element Methods in Solid Mechanics
```

Comment:
Move reference (top of file?) and cite with a shorthand

Reply:
All references moved to module doc string, referencing in inline comments reworked.